.github/workflows/coverage.yml

```diff
@@ -25,10 +25,9 @@ jobs:
         sed -i.bak '/examples/d' Cargo.toml
     - name: Install cargo-llvm-cov
       uses: taiki-e/install-action@cargo-llvm-cov
+    - name: Generate coverage report (features enabled)
```

Comment:
This can take an hour (and it only expects one coverage report to be posted).

Reply:
Not sure how we should actually resolve this lol.

Reply:
This one snuck under the radar in our previous refactor (where we introduced everything the test casing).